staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cached_token_authenticator_test.go

```diff
@@ -547,3 +550,107 @@ func withAudit(ctx context.Context) context.Context {
 	ac.Event = &auditinternal.Event{Level: auditinternal.LevelMetadata}
 	return ctx
 }
+
+func TestUnsafeConversions(t *testing.T) {
+	t.Parallel()
+
+	const size = 1024 // needs to be large to force allocations
```

Comment:
I'd feel a bit more confident if size was, say, random between 1024 and 2048.

Reply:
[Done](https://github.com/kubernetes/kubernetes/compare/50b841d46d7ceecb81c33b087ced6c41476ed36f..58ff7aab35333812213f49bbae05ded58aa50cf9?w=1).